### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**
UI Fixes

### CODE REVIEW

File: lib/components/flat_button_long.dart [hunk 1]
```diff
@@ -26,7 +26,7 @@ class FlatButtonLong extends StatelessWidget {
                 width: 17,
                 height: 17,
                 child: const CircularProgressIndicator(
-                  // color: AppColors.white,
```
Comment (gguijarro-c-chwy) [inline lib/components/flat_button_long.dart:29]: image
This was causing the indicator to not be visible

File: lib/screens/profile_screens/guardians/guardians_tabs/components/my_guardians_view.dart [hunk 2]
```diff
@@ -42,7 +42,10 @@ class MyGuardiansView extends StatelessWidget {
                         onPressed: () {
                           BlocProvider.of<GuardiansBloc>(context).add(OnRemoveGuardianTapped(e));
                         },
-                        child: Text(state.areGuardiansActive ? '' : 'Remove')),
+                        child: Text(
```
Comment (gguijarro-c-chwy) [inline lib/screens/profile_screens/guardians/guardians_tabs/components/my_guardians_view.dart:45]: it looked disabled. Not it looks active

File: lib/screens/authentication/login_screen.dart [hunk 3]
```diff
@@ -27,8 +28,14 @@ class LoginScreen extends StatelessWidget {
...
             mainAxisAlignment: MainAxisAlignment.center,
             children: [
-              Text('Recover your funds ', style: Theme.of(context).textTheme.subtitle2),
-              const Text('here'),
+              Text(
+                'Recover your funds ',
+                style: Theme.of(context).textTheme.subtitle2,
+              ),
+              Text(
+                'here',
```
Comment (gguijarro-c-chwy) [inline lib/screens/authentication/login_screen.dart:36]: image

File: lib/screens/authentication/import_key/import_key_screen.dart [hunk 4]
```diff
@@ -63,12 +63,11 @@ class ImportKeyScreen extends StatelessWidget {
                                   onPressed: () async {
                                     final clipboardData = await Clipboard.getData('text/plain');
                                     final clipboardText = clipboardData?.text ?? '';
-                                    final newText =
-                                        textController.value.replaced(textController.selection, clipboardText).text;
-                                    textController.text = newText;
+                                    textController.text = clipboardText;
                                     // ignore: use_build_context_synchronously
-                                    BlocProvider.of<ImportKeyBloc>(context)
```
Comment (gguijarro-c-chwy) [inline lib/screens/authentication/import_key/import_key_screen.dart:70]: this code was preventing the first tap from copying the text into the field.

File: lib/screens/authentication/login_screen.dart (hunk 3 above)
Comment (gguijarro-c-chwy) [reply lib/screens/authentication/login_screen.dart:36]: should this even be there? Dont we recover an account after creating an account?

File: lib/screens/authentication/recover/recover_account_search/recover_account_screen.dart [hunk 5]
```diff
@@ -104,20 +109,9 @@ class _RecoverAccountScreenState extends State<RecoverAccountScreen> {
...
             Navigator.pop(context);
-            // TODO(n13): This needs to actually initialize the recovery!
-
-            // TODO here:
-            // maybe pop the dialog, then show a progress indicator (this takes 6 seconds! an eternity!)
-            // then show the next screen, recoverAccount details
-            // or it could show a "recovery initiated successfully" dialog, then on OK on that move on to Routes.recoverAccountDetails
-            // anyway it needs to end up in Routes.recoverAccountDetails so the user can share the link etc (that part already works)
-            final address = accountService.currentAccount.address;
-            await polkadotRepository.recoveryRepository.initiateRecovery(rescuer: address, lostAccount: lostAccount);
```
Comment (gguijarro-c-chwy) [inline lib/screens/authentication/recover/recover_account_search/recover_account_screen.dart:117]: making network alls from the UI? Shame! 
I removed this from the UI and moved to the bloc. 

The bloc also handles loading, success and error message now.

Comment (gguijarro-c-chwy) [reply lib/screens/authentication/recover/recover_account_search/recover_account_screen.dart:117]: image

File: lib/screens/authentication/login_screen.dart (hunk 3 above)
Comment (n13) [reply lib/screens/authentication/login_screen.dart:36]: Yes this should not be there

File: lib/screens/authentication/recover/recover_account_search/recover_account_screen.dart (hunk 5 above)
Comment (n13) [reply lib/screens/authentication/recover/recover_account_search/recover_account_screen.dart:117]: Yep that was the idea thank you! :)

File: lib/screens/authentication/import_key/import_key_screen.dart (hunk 4 above)
Comment (n13) [reply lib/screens/authentication/import_key/import_key_screen.dart:70]: No, you can't do that. 

I am not sure what the problem was with the existing code, but I put that code in there to make copy / paste work correctly. 

Copy paste means that whatever is currently selected in the text field is replaced by the pasted text. 

This is basics, it needs to work this way. 

It's not just about the button but also about hitting "paste" from the text editing overlay. 

For example let's say the text is 

"12345foobar"

I go outside the app and copy the text 6789 to the clipboard. . 

Then I go back and select "foobar", then hit paste - now the new string should 123456789

If we just replace text with clipboard then we get only 6789 as the new text. 

I will check into the first tap issue.

Comment (n13) [reply lib/screens/authentication/import_key/import_key_screen.dart:70]: It's how any text editor works, so we need to work the same way

It's going to be maybe rare that someone wants to copy paste parts of a string but from a UX perspective ... copy paste rules must be followed.

**Pull request merged**